src/main/java/org/opentestsystem/ap/imrt/common/model/BaseItem.java

```diff
@@ -0,0 +1,156 @@
+import javax.validation.constraints.NotNull;
+import java.time.Instant;
+import java.util.List;
+
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@Table(name = "item")
+@EntityListeners(AuditingEntityListener.class)
+@DiscriminatorColumn(name = "classification")
+public abstract class BaseItem extends BaseEntity {
+    @Id
+    @Column
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer key;
+
+    @NotBlank
+    private String id;
+
+    private String subject = "";
+
+    private String grade = "";
+
+    @NotBlank
+    private String workflowStatus;
+
+    @NotBlank
+    @Column(name = "item_type")
+    private String itemType;
+
+    @Column(name = "dok")
+    private String depthOfKnowledge = "";
+
+    @NotBlank
+    private String itemCreatedBy;
+
+    @Basic
+    private Instant itemCreatedAt;
+
+    @Type(type = "jsonb")
+    @Column(columnDefinition = "jsonb")
+    @NotNull
+    private org.opentestsystem.ap.common.model.Item itemJson;
+
+    @OneToMany
+    @JoinColumn(name = "item_key")
+    @OrderBy("type ASC")
+    private List<StandardId> standardIds;
+
+    public List<StandardId> getStandardIds() {
+        return standardIds;
+    }
+
+    public void setStandardIds(final List<StandardId> standardIds) {
+        this.standardIds = standardIds;
+    }
+
+    public Integer getKey() {
+        return key;
+    }
+
+    public void setKey(final Integer key) {
+        this.key = key;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(final String id) {
+        this.id = id;
+    }
+
+    public String getSubject() {
+        return subject;
+    }
+
+    public void setSubject(final String subject) {
+        this.subject = subject;
+    }
+
+    public String getGrade() {
+        return grade;
+    }
+
+    public void setGrade(final String grade) {
+        this.grade = grade;
+    }
+
+    public String getWorkflowStatus() {
+        return workflowStatus;
+    }
+
+    public void setWorkflowStatus(final String workflowStatus) {
+        this.workflowStatus = workflowStatus;
+    }
+
+    public String getItemType() {
+        return itemType;
+    }
+
+    public void setItemType(final String itemType) {
+        this.itemType = itemType;
+    }
+
+    public String getDepthOfKnowledge() {
+        return depthOfKnowledge;
+    }
+
+    public void setDepthOfKnowledge(final String depthOfKnowledge) {
+        this.depthOfKnowledge = depthOfKnowledge;
+    }
+
+    public String getItemCreatedBy() {
+        return itemCreatedBy;
+    }
+
+    public void setItemCreatedBy(final String itemCreatedBy) {
+        this.itemCreatedBy = itemCreatedBy;
+    }
+
+    public Instant getItemCreatedAt() {
+        return itemCreatedAt;
+    }
+
+    public void setItemCreatedAt(final Instant itemCreatedAt) {
+        this.itemCreatedAt = itemCreatedAt;
+    }
+
+    public org.opentestsystem.ap.common.model.Item getItemJson() {
```

Comment:
I don't grok this but i don't really have to ...

Reply:
Yeah, so we have the item json that is created by IAT and IMRT's item.  This is the Java representation of the item.json that is created by IAT.  The naming is going to be a bit tricky within IMRT because we have our "item" and IAT's item but at a base level they are the same.

Reply:
Makes sense to me - this is just IMRT's representation of an Item.